packages/mobile/src/fiatExchanges/ExternalExchanges.tsx

```diff
@@ -0,0 +1,111 @@
+import ListItem from '@celo/react-components/components/ListItem'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import variables from '@celo/react-components/styles/variables'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { FiatExchangeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import AccountNumber from 'src/components/AccountNumber'
+import BackButton from 'src/components/BackButton.v2'
+import { EXCHANGE_PROVIDER_LINKS } from 'src/config'
+import i18n from 'src/i18n'
+import LinkArrow from 'src/icons/LinkArrow'
+import { emptyHeader } from 'src/navigator/Headers.v2'
+import { navigateToURI } from 'src/utils/linking'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+export const externalExchangesScreenOptions = () => {
```

Comment:
I believe the pattern we are trying to move towards is including the navigation options as a property of the component. See the Welcome screen pattern as an example

Reply:
For now will keep this consistent with the rest of the fiat exchange-related flows but agree that we should have something consistent across the codebase